packages/dataviews/src/layouts/list/index.tsx

```diff
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import clsx from 'clsx';
+// TODO: use the @wordpress/components one once public
```

Comment:
We can, but as I split the whole process on several PRs, this PR focuses on exporting the v2 as stable. 

Also, as recently discussed with Lena, we will try to initially avoid exporting types — instead of using the type from `@wordpress/components`, we will use `React.ComponentProps`, `ReturnType<>` and `Parameters<>` to extract the type from the exported components and hooks. 

I will update the code comment.

Reply:
Sounds good!